pwndbg/heap/__init__.py

```diff
@@ -5,31 +5,33 @@
 
 current = None
 
-main_arena = pwndbg.gdblib.config.add_param("main-arena", "0", "&main_arena for heuristics", "heap")
+main_arena = pwndbg.gdblib.config.add_param(
+    "main-arena", "0", "&main_arena for heuristics", scope="heap"
```

Comment:
Just like we have `theme.add_param` and `theme.add_color_param`, maybe we should have some `add_heap_param` function so we don't need to type out the scope everywhere.

Reply:
Sure, I'll add `add_heap_param` for it.